pkgs/by-name/wi/windsurf/package.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  vscode-generic,
+  fetchurl,
+  nixosTests,
+  commandLineArgs ? "",
+  useVSCodeRipgrep ? stdenv.hostPlatform.isDarwin,
+}:
+let
+  info =
+    (lib.importJSON ./info.json)."${stdenv.hostPlatform.system}"
+      or (throw "windsurf: unsupported system ${stdenv.hostPlatform.system}");
```

Comment:
I wonder if this is really required since you've clearly defined the `meta.platforms`.

Reply:
If I don't add this, ci may fails

Reply:
Fair enough, good to know :)